modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -87,19 +88,22 @@ def config_template(self):
             return IgniteClientConfigTemplate()
         return IgniteServerConfigTemplate()
 
-    def __home(self, version=None):
+    def __home(self, version=None, project=None):
         """
         Get home directory for current spec.
         """
+        project = self.project if project is None else project
```

Comment:
May be better is `project = project if project else self.project`, as at code below with version?

Reply:
Fixed. Thanks